package.json

```diff
@@ -37,15 +38,18 @@
     "eslint-plugin-svelte3": "3.0.0",
     "jest": "^26.6.3",
     "lerna": "3.22.1",
+    "node-sass": "^5.0.0",
```

Comment:
@wlach Yup everything was checked in, sass was removed and node-sass installed. Your error message https://gist.github.com/wlach/8c7fc5abd046537f1d7348bfd8d55203#file-gistfile1-txt-L31 indicates that you still have `sass` in `node_modules` though, you probably need to do a refresh `npm install`

Reply:
As mentioned, I did a fresh checkout and am still seeing the error. I definitely don't have `sass` in `node_modules`:

```
wlach@antwerp glean-dictionary % ls node_modules/sass-*
node_modules/sass-graph:
CHANGELOG.md		package.json		sass-graph.js
bin			parse-imports.js
node_modules		readme.md

node_modules/sass-loader:
CHANGELOG.md	README.md	node_modules
LICENSE		dist		package.json
```

Does it really work for you if you do a fresh checkout? If so I can't imagine what the problem might be...

Reply:
Nevermind, just figured it out. I had a copy of `sass` installed globally. I wonder if there's a way to tell glean-dictionary to prefer `node-sass` in that case?

Reply:
> Nevermind, just figured it out. I had a copy of `sass` installed globally. I wonder if there's a way to tell glean-dictionary to prefer `node-sass` in that case?

Just googled and I'm not seeing any easy solutions :/ Let's table this for now, I don't think it should block merging (unless you have easy ideas).